src/find-specs.js

```diff
@@ -77,6 +77,7 @@ const hasUntrackedURL = ({spec: url}) => !specs.find(s => s.nightly.url.startsWi
                                             ));
 const hasUnknownTrSpec = ({spec: url}) => !specs.find(s => s.release && trimSlash(s.release.url) === trimSlash(url)) && !specs.find(s => hasMoreRecentLevel(s,url));
 
+const composeFilters = (f1, f2) => value => f1(value) || f2(value);
```

Comment:
compose is a bit generic (not clear if it is `or` or `and`) - maybe `eitherFilters` or `unionFilters`?

Reply:
True. Code now uses `eitherFilter`.